docs/support/tutorials/services-for-courses.md

```diff
@@ -0,0 +1,49 @@
+# CSC Services for Courses
+
+This guide will explain how to use the course project type with CSC's
+high-performance computing (HPC) and cloud services.
+
+## Prerequisites
+
+Staff members from higher education institutions and state research institutes can create a MyCSC course project. What you will need is:
+
+   1. [A CSC account](../../accounts/how-to-create-new-user-account.md)
+   2. [A MyCSC course project](../../accounts/how-to-create-new-project.md)
+   3. [A Service activated](../../accounts/how-to-add-service-access-for-project.md)
+
+## Available services
```

Comment:
Should the list (just a list) be right in the first paragraph.  Perhaps even with info which one needs  a project, and which does not. This is then actually a section with guidelines, service specific information etc.

Reply:
Perhaps we could also there add the short description what is the use case / strengths for each service so users can pick the right one